pkg/controller/compute/gke_test.go

```diff
@@ -124,6 +127,41 @@ func TestSyncClusterGetError(t *testing.T) {
 	assertResource(g, r, expectedStatus)
 }
 
+func TestSyncErroredCluster(t *testing.T) {
+	g := NewGomegaWithT(t)
```

Comment:
https://github.com/crossplaneio/crossplane/issues/452

Just as a heads up, we're trying to move away from using Gomega. I'm happy to accept this new Gomega test given that it matches the rest of the tests in this package, but for new packages I'd prefer to use the patterns suggested in the above issue.

Reply:
Thanks for the heads up, I'll check.